src/coffee/run.coffee

```diff
@@ -89,9 +89,7 @@ module.exports = class
   @run: (argv) ->
     
     _consoleWarnAllResults = (result) ->
-      # print out full response array by passing util.inspect
-      # with maxArrayLength (default is 100)
-      console.warn util.inspect(result, { maxArrayLength: null })
+      console.warn result
```

Comment:
Remove `util.inspect` because its output can 't be depended on ([see the link](https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_inspect_object_options)).

Reply:
What do you mean with it cant be depended on?

Reply:
From the Node docs, `Util.inspect` output can change anytime without public release or update and from the issues, it shows that `Util.inspect` is not doing what was expected or needed (to output all the result).

Reply:
Okay, but if the default is `100` then this change means that we will not show anything between entry `101` and `500`, right?

Reply:
Yes.

Reply:
Ok, I understand and this looks very dangerous to me. I looked at the git history and found [this PR](https://github.com/sphereio/sphere-node-product-csv-sync/pull/227) which introduced the change to using `util.inspect`.

The `_consoleWarnAllResults` seems to also be used in these places [1](https://github.com/sphereio/sphere-node-product-csv-sync/blob/faa1eead968e3f4d11c3c7c6b744becb1e8cdbd9/src/coffee/run.coffee#L187), [2](https://github.com/sphereio/sphere-node-product-csv-sync/blob/faa1eead968e3f4d11c3c7c6b744becb1e8cdbd9/src/coffee/run.coffee#L352), and [3](https://github.com/sphereio/sphere-node-product-csv-sync/blob/faa1eead968e3f4d11c3c7c6b744becb1e8cdbd9/src/coffee/run.coffee#L401). 

As the PR has not introduced an output `log file` for those commands, that means we will not output all the logging results for them. 

Seems a safer bet to me to keep `util.inspect()` here. The other alternative would be to also save outputted result logs to files in the three other commands.

Does it make sense? WDYT?

Reply:
Sure, it's okay for now to avoid breaking the other modules that use the `_consoleWarnResults` but in future, it will be better to avoid the `Util.inspect()`.